src/wp-includes/blocks/social-link.php

```diff
@@ -62,10 +62,10 @@ function render_block_core_social_link( $attributes, $content, $block ) {
 	$processor = new WP_HTML_Tag_Processor( $link );
 	$processor->next_tag( 'a' );
 	if ( $open_in_new_tab ) {
-		$processor->set_attribute( 'rel', esc_attr( $rel ) . ' noopener nofollow' );
+		$processor->set_attribute( 'rel', trim( $rel . ' noopener nofollow' ) );
```

Comment:
Why is `$rel` this no longer escaped?

Reply:
cc @dmsnell

Reply:
nothing in the HTML API should be escaped, as the Tag Processor performs the necessary escapes. leaving them in will lead to double-escaping and corruption.